### ISSUE TITLE
Errors on default project file

### ISSUE THREAD

**Issue by Hokome** [YarnSpinnerTool/YarnSpinner-Unity#176]
**What is the current behavior?**
Getting errors when clicking on default project file. (only way to fix I found was to add variables)

**Please provide the steps to reproduce, and if possible a minimal demo of the problem**:

- Install Yarn Spinner
- Create Yarn Project file
- Click on the created project file

**Please tell us about your environment:**
  
  - Yarn Spinner Version: 2.1.0
  - Unity Version: 2021.3.0f

**Other information** 

Full errors and stack traces: [https://pastebin.com/sfjZU9wd](url)

**Comment by desplesda** [comment]
This is possibly a duplicate of #168, but I'll leave it open so it can be checked after #171 has been merged.

**Comment by desplesda** [comment]
Closed as duplicate of #168.
